data/json/effects.json

```diff
@@ -548,7 +548,7 @@
     "rating": "bad",
     "miss_messages": [ [ "Your lungs burn from the smoke.", 1 ] ],
     "harmful_cough": true,
-    "max_duration": 600,
+    "max_duration": "10m",
```

Comment:
Whitespace here? Or is it not required?

Reply:
Not required but spaces are the style most of the JSON uses.